ush/python/pygfs/task/aero_emissions.py

```diff
@@ -37,48 +44,446 @@ def __init__(self, config: Dict[str, Any]) -> None:
         """
         super().__init__(config)
 
-        local_variable = "something"
+        current_datetime = add_to_datetime(
+            self.task_config.PDY,
+            to_timedelta(f"0H"),
+        )
+        nforecast_hours = self.task_config["FHMAX_GFS"]
+        nforecast_days = nforecast_hours // 24 + 1
+        forecast_dates = [current_datetime + timedelta(days=i) for i in range(nforecast_days)]
 
         localdict = AttrDict(
-            {'variable_used_repeatedly': local_variable}
+            {"cdate": current_datetime, "nforecast_days": nforecast_days, "forecast_dates": forecast_dates}
         )
 
         # Extend task_config with localdict
         self.task_config = AttrDict(**self.task_config, **localdict)
 
+        # Read the aero_emission.yaml file for common configuration
+        logger.info(
+            f"Read the prep_emission configuration yaml file {self.task_config.PREP_EMISSION_CONFIG}"
+        )
+        self.task_config.aero_emission_yaml = parse_j2yaml(
+            self.task_config.PREP_EMISSION_CONFIG, self.task_config
+        )
+        logger.debug(
+            f"aero_emission_yaml:\n{pformat(self.task_config.aero_emission_yaml)}"
+        )
+
     @staticmethod
     @logit(logger)
-    def initialize() -> None:
-        """Initialize the work directory
+    def initialize(aero_emission_yaml: Dict) -> None:
+        """Initialize the work directory by copying all the common fix data
+
+        Parameters
+        ----------
+        aero_emission_yaml: Dict
+            Fully resolved aero_emissions.yaml dictionary
+
+         Returns
+        -------
+        None
         """
+        logger.info("Copy Static Data to run directory")
+
+        print(aero_emission_yaml)
```

Comment:
```suggestion
```

Or make it into a logger entry.

Reply:
sorry was debugging things and forgot to remove this